src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.cs

```diff
@@ -24,10 +24,12 @@ public static void WebSocket_Http_RequestReply_BinaryStreamed()
         try
         {
             // *** SETUP *** \\
-            binding = new NetHttpBinding();
+            binding = new NetHttpBinding()
+            {
+                MaxReceivedMessageSize = ScenarioTestHelpers.DefaultMaxReceivedMessageSize,
+                MaxBufferSize = ScenarioTestHelpers.DefaultMaxReceivedMessageSize,
```

Comment:
Nit: probably want a separate const for DefaultMaxBufferSize rather than reusing the max received message size.  Consider for future cleanup -- not needed for merge.